python/GafferArnoldUI/ArnoldAttributesUI.py

```diff
@@ -572,6 +576,30 @@ def __volumeSummary( plug ) :
 
 		],
 
+		"attributes.velocityScale" : [
+
+			"description",
+			"""
+			Scale the vector used in vdb  motion blur computation.
```

Comment:
Scale**s** the vector (extra s). The prevailing style for descriptions is to write them such that you could prefix them with "This plug" or "This plug is" and they would form a complete sentence. No need to expose users to the weird imperative tense we use in commits. While I'm nitpicking, there's an extra space after vdb, and VDB should be capitalised.